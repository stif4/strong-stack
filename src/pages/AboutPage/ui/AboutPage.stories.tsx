import React from 'react';
import { Meta, StoryFn } from '@storybook/react';
import AboutPage from './AboutPage';

export default {
    title: 'pages/AboutPage',
    component: AboutPage,
} satisfies Meta<typeof AboutPage>;

const Template: StoryFn<typeof AboutPage> = (args) => <AboutPage {...args} />;

export const Normal = Template.bind({});
Normal.args = {};
