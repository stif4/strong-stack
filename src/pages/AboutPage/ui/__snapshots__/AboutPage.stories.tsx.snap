// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages/AboutPage Normal smoke-test 1`] = `
<div class="app dark">
  <div>
    О сайте
  </div>
</div>
`;
